### PR TITLE
Activity questions

### PR DESCRIPTION
**Pull request by AmritSidhu**
https://trello.com/c/reemF8gj/159-checks-on-identity-questions-in-tool

### CODE REVIEW

File: app/controllers/activity_questions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+  def activity_3
+    if request.post?
+      if !params[:activity_time_period]
+        @errors[:activity_time_period] = 'You must answer the question'
+      end
+    end
+
+    if request.get? || !@errors.empty?
+      @shared = Form.new('shared')
```
Comment (AmritSidhu) [inline app/controllers/activity_questions_controller.rb]: Is this needed?

File: app/controllers/activity_questions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  def activity_2
+    if request.post?
+      if !params[:org_activity_found]
+        @errors[:org_activity_found] = 'You must answer the question'
+      end
+    end
+
+    if request.get? || !@errors.empty?
+      @shared = Form.new('shared')
+      @form = Form.new('assessments')
```
Comment (AmritSidhu) [inline app/controllers/activity_questions_controller.rb]: Is this needed?

File: app/controllers/activity_questions_controller.rb (hunk 1 above)
Comment (galund) [reply app/controllers/activity_questions_controller.rb]: Probably not; I don't think I mind it being there. It's quicker to copy-paste if we don't worry about minor differences between the handlers like this one, and if we do manage a further refactoring, then we'd just make shared stuff be present in all the templates anyway.

**Comment by galund** [comment]
@laurencecodes adding you FYI, we're gonna land this anyway, but let us know if you spot anything weird :-)

**Pull request merged**